clang/include/clang/3C/3CInteractiveData.h

```diff
@@ -16,18 +16,26 @@
 #include "clang/3C/ConstraintVariables.h"
 #include "clang/3C/PersistentSourceLoc.h"
 
-// Source info and reason for each wild pointer.
-class WildPointerInferenceInfo {
+// Source info and reason
+class Feedback {
```

Comment:
Just calling this `Feedback` feels a little too broad. Is there some more specific name you could give it?

Reply:
It's used in the root cause output, so how about `RootCauseDiagnostic`? Or I can use the original name. I renamed it when I thought it was going to be used more, but I ended up with `ReasonLoc` for broad use instead.

Reply:
The original name wasn't great, so let's not revert to that. `RootCauseDiagnostic` sounds fine to me.